book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -106,12 +106,12 @@ Fast-forward
 
 You'll notice the phrase ``fast-forward'' in that merge.
 Because the commit `C4` pointed to by the branch `hotfix` you merged in was directly ahead of the commit `C2` you're on, Git simply moves the pointer forward.
-To phrase that another way, when you try to merge one commit with a commit that can be reached by following the first commit's history, Git simplifies things by moving the pointer forward because there is no divergent work to merge together -- this is called a ``fast-forward.''
+To phrase that another way, when you try to merge one commit with a commit that can be reached by following the first commit's history, Git simplifies things by moving the pointer forward because there is no divergent work to merge together -- this is called a ``fast-forward''.
```

Comment:
This was written to the [AP style](https://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html), where punctuation goes inside quotation marks. The exceptions we make are when we're literally quoting a bit of UI that doesn't include the punction, like this:

> Open the "File" menu, then click "Save".

Otherwise, periods go inside the quotes.

Reply:
Oh! I see, we follow different style. @ben I think it would be nice, if could compile all these information at one place and include it with project repository, may be Github wiki would be a good idea. And thanks for the article link.